ui/README.md

```diff
@@ -0,0 +1,40 @@
+# The renovate interface
+
+The renovate interface is the desktop, mobile and iframe_api interfaces from ngeo that uses the ngeo directly (renovate).
+
+## Development
+
+The server application should be started previously, you can start it by running:
```

Comment:
I would say "see the README at the root of the project".
To not maintain twice the same doc (and you don't mention the secret here, that's already a possible "issue")

Reply:
Updated, do you think that's good like that?

Reply:
Yes thanks :-)